examples/utils/webpack-fluid-loader/src/loader.ts

```diff
@@ -112,14 +117,14 @@ function makeSideBySideDiv(divId: string) {
 /**
  * Create a loader with WebCodeLoader and return it.
  */
-async function createWebLoader(
+async function createWebLoaderProps(
```

Comment:
"web loader" is meaningless, as long as we're renaming let's drop "web"
```suggestion
async function createLoaderProps(
```